pkg/clusterapi/aws/actuator_test.go

```diff
@@ -187,10 +187,20 @@ func TestCreateMachine(t *testing.T) {
 			actuator.clientBuilder = func(kubeClient kubernetes.Interface, mSpec *clustopv1.MachineSetSpec, namespace, region string) (Client, error) {
 				return mockAWSClient, nil
 			}
+			var userDataGeneratorCalled bool
```

Comment:
`userDataGeneratorCalled := false` instead of implicitly relying on false being the default.